src/freenet/node/useralerts/UserAlertManager.java

```diff
@@ -158,6 +158,9 @@ public int compare(UserAlert a0, UserAlert a1) {
 			int classHash1 = a1.getClass().hashCode();
 			if(classHash0 > classHash1) return 1;
 			else if(classHash0 < classHash1) return -1;
+			// Then go by sent time
```

Comment:
The comment says sent time; the getUpdatedTime() implementation says receivedTime.